src/Pool.sol

```diff
@@ -453,6 +475,25 @@ abstract contract Pool {
         return subSenders[senderAddr][subSenderId].amtPerSec;
     }
 
+    /// @notice Sets the fraction of received funds to be dripped by the sender.
+    /// @param sender The updated sender
+    /// @param dripsFraction The fraction of received funds to be dripped.
+    /// Must be a value from 0 to `DRIPS_FRACTION_MAX` inclusively,
+    /// where 0 means no dripping and `DRIPS_FRACTION_MAX` dripping everything.
+    function _setDripsFraction(Sender storage sender, uint32 dripsFraction) internal {
+        require(dripsFraction <= DRIPS_FRACTION_MAX, "Drip fraction too high");
+        sender.dripsFraction = dripsFraction;
+    }
+
+    /// @notice Gets the fraction of received funds to be dripped by the provided user.
+    /// @param userAddr The address of the user
+    /// @return dripsFraction The fraction of received funds to be dripped.
+    /// A value from 0 to `DRIPS_FRACTION_MAX` inclusively,
+    /// where 0 means no dripping and `DRIPS_FRACTION_MAX` dripping everything.
+    function getDripsFraction(address userAddr) public view returns (uint32 dripsFraction) {
```

Comment:
```suggestion
    function dripsFraction(address userAddr) public view returns (uint32 dripsFraction) {
```
In standard Solidity coding style it would be without a `get`

Reply:
Ok, should we keep for now `getDripsFraction` for consistency with other methods? IMO with addition of drips and possible extraction of sender config to calldata we'll need to rename or remove a few methods soon anyway.

Reply:
Okay 👍